User-Help/CodidactCommentFlagTexts.md

```diff
@@ -0,0 +1,31 @@
+## Flag reasons and texts
+
+**Why are you flagging this comment?**
+
+Please select a flag reason...
+
+**Code of Conduct violation**
+
+This comment is rude, disrespectful, or otherwise violates our [Code of Conduct](/policy/code-of-conduct).
+
```

Comment:
Can you add the "Spam" flag? This would likely fall under the CoC but it's helpful for moderators to be able to see at a glance whether a user is being a jerk or spamming.